<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="rd_boys">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Nam</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rd_girl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>69</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Nữ</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_gender">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Giới tính</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
